CHANGELOG.md

```diff
@@ -107,6 +107,8 @@ Biome now scores 97% compatibility with Prettier and features more than 180 lint
 
 - Fix [#1106](https://github.com/biomejs/biome/issues/1106). Fix invalid formatting of single bindings when Arrow Parentheses is set to "AsNeeded" and the expression breaks over multiple lines. [#1449](https://github.com/biomejs/biome/pull/1449) Contributed by @faultyserver
 
+- Fix [#1172](https://github.com/biomejs/biome/issues/1172). Fix placement of line comment after function expression parentheses, they are now attached to first statement in body. Contributed by @kalleep
```

Comment:
I just saw that a new release was made for biome, but there is no _Unreleased_ section created.

Reply:
After a fresh release, the first to arrive with a PR needs to add the unreleased section. You can find a template in the contributing document

Reply:
fixed in [df3c813](https://github.com/biomejs/biome/pull/1485/commits/df3c8138975a9139065724e84331009511b9feea)